platforms/extensibility/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/tasks/ValidatePluginsPart2IntegrationTest.groovy

```diff
@@ -602,7 +602,8 @@ class ValidatePluginsPart2IntegrationTest extends AbstractIntegrationSpec implem
         supportedType | value
         'Integer'     | 'Integer.valueOf(0)'
         'String'      | '"foo"'
-        'Enum'        | 'Letter.A'
+        'Letter'      | 'Letter.A'
+        'Enum'        | 'Letter.A' // This is how we checked before
```

Comment:
❓ This comment in the code makes no sense to me; what's before here, outside the context of this PR?

Reply:
The original check would only work if you declared the property as:
```
@Nested Map<Enum, Options>
```
I'll make this clearer in the comment.